<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressTool</class>
 <widget class="QDialog" name="AddressTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_gridLayout">
     <item row="2" column="0">
      <widget class="QCheckBox" name="m_reverseVertCheck">
       <property name="text">
        <string>Reverse vertically</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Colour</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QSpinBox" name="m_addressSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="m_redBtn">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="m_blueBtn">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="m_blackBtn">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QCheckBox" name="m_reverseHorizCheck">
       <property name="text">
        <string>Reverse Horizontally</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddressTool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddressTool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_redBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddressTool</receiver>
   <slot>slotChangeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>785</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_blueBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddressTool</receiver>
   <slot>slotChangeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>793</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_blackBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddressTool</receiver>
   <slot>slotChangeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>807</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChangeColor()</slot>
 </slots>
</ui>
